src/DevBetterWeb.Web/Pages/Leaderboard/Index.cshtml

```diff
@@ -35,20 +36,42 @@
         </table>
     </div>
 
-    <div class="col-6">
-         <div class="card">
-             <div class="card-header">Available Books</div>
-             <div class="card-body">
-                 <strong>There are @Model.Books.Count books you can read to contribute to your leaderboard standings:</strong>
-                 <br />
-                 <br />
-                 @foreach(var book in Model.Books)
-                 {
-                     <a asp-page="BookDetails" asp-route-bookId="@book.Id"> @book.ToString()</a>
-                     <br />
-                 }
-             </div>
-         </div>
+    <div class="col-7">
+        <table class="table table-sm table-striped" id="leaderboardBooksTable">
+            <thead>
+                <tr>
+                    <th>Rank</th>
+                    <th>Book</th>
+                    <th></th>
+                    <th></th>
+                    <th>Read by...</th>
+                </tr>
+            </thead>
+            <tbody>
+                @foreach (var book in Model.Books)
+                {
+                    <tr>
+                        <td>@(bookrank++)</td>
+                        <td>
+                            <a asp-page="BookDetails" asp-route-bookId="@book.Id"> @book.ToString()</a>
+                        </td>
+                        <td>
+                            <a href="@book.PurchaseUrl" target="_blank"><i class="fas fa-shopping-cart"></i></a>
+                        </td>
+                        <td></td>
+                        <td>
+                            @book.MembersWhoHaveRead!.Count 
+                            @if(book.MembersWhoHaveRead.Count == 1)
+                            {
+                                <a>Member</a>
```

Comment:
Why did you wrap this in <a> tags if it isn't a link? If you just needed Razor to understand it's HTML content not C# code, use a <span> or read this thread:
https://stackoverflow.com/questions/27284734/render-a-string-as-html-in-c-sharp-razor (don't use Html.Raw)